src/App/App.tsx

```diff
@@ -39,6 +40,10 @@ function App() {
 								</Typography>
 							</Container>
 						</Route>
+						<Route path="/searchUsers" exact>
+							{/* Search Users page */}
+								<SearchUsers api={api}></SearchUsers>
```

Comment:
Let's make this self closing - it doesn't have or accept children props at the moment

```suggestion
								<SearchUsers api={api} />
```